atlascharts/sunburst.js

```diff
@@ -28,23 +28,28 @@ define(["d3", "./chart"], function (d3, Chart) {
 		getTipDirection(d) {
 			return("n");
 		}
-		
-		getTipOffset(d, arc) {
+
+		getTipOffset(d, arc, options) {
 			const bbox = event.target.getBBox();
 			const arcCenter = arc.centroid(d);
+			const { tipPosition = 'bottom' } = options;
 			let tipOffsetX = Math.abs(bbox.x - arcCenter[0]) - (bbox.width/2)
-			let tipOffsetY = Math.abs(bbox.y - arcCenter[1]);
-			return([tipOffsetY-10,tipOffsetX]);
+			let tipOffsetY;
+			tipPosition === 'bottom' && (tipOffsetY = Math.abs(bbox.y - arcCenter[1]) - 10);
+			tipPosition === 'top' && (tipOffsetY = Math.abs(bbox.y + arcCenter[1]) + 10);
+
+			return [tipOffsetY, tipOffsetX];
 		}
-		
+
 		render(data, target, width, height, chartOptions) {
-			
+
 			super.render(data, target, width, height, chartOptions);
 
 			const defaultOptions = {
 				tooltip: (d) => {
 					return `<div>No Tooltip Set</div>`
-				}
+				},
+				tipPosition: 'bottom', // available options bottom|top
```

Comment:
I would not say that the current default position is bottom, the arrow goes from nearly middle of a slice. And the ballon is not below the error (which is typically assumed as tooltip-location=bottom):
![image](https://user-images.githubusercontent.com/10010071/54313999-e29cd380-45b0-11e9-8559-cbb1f8285f60.png)